Source/EasyNetQ/Producer/DefaultRpc.cs

```diff
@@ -260,7 +260,7 @@ CancellationToken cancellationToken
             return advancedBus.Consume<TRequest>(
                 queue,
                 (m, i, c) => RespondToMessageAsync(responder, m, c),
-                c => c.WithPrefetchCount(configuration.PrefetchCount)
+                c => c.WithPrefetchCount(configuration.PrefetchCount).WithPriority(configuration.Priority)
```

Comment:
Are the priority values 0 and "not set" any different? Is there a situation where the queues already created will throw an PRECONDITION_FAILED error (I do not remember the exact error name)?

Reply:
My changed ResponderConfiguration class sets priority of ConsumerConfiguration class which is initialized with 0 also. so I dont think there is any change. by default it will set 0 to 0.
https://github.com/EasyNetQ/EasyNetQ/blob/b7936a70a1b08e1666a208faaf1378d06fc41a09/Source/EasyNetQ/Consumer/IConsumerConfiguration.cs

Reply:
@Pliner Any thoughts?